### ISSUE TITLE
locate-error calls without parameters fails; this causes incorrect error messages.

### ISSUE THREAD

**Issue by zellerin** [melisgl/mgl-pax#33]
Locate-error function docstring suggests that format-and-args may be empty, and in some calls it is. However, destructuring-bind in document-documentable/with-document-context then fails.

Example:
```
(mgl-pax:defsection @test
    ()
  (foobar foobar))

(mgl-pax:document @test)
```

gives
```
Error while parsing arguments to DESTRUCTURING-BIND:
  too few elements in
    ()
  to satisfy lambda list
    (FORMAT-CONTROL &REST FORMAT-ARGS):
  at least 1 expected, but got 0
```

Simple fix: change in locate-error function
```
:message format-and-args
```
to
```
:message (or format-and-args '(""))
```

The error now reads correctly
```
Could not locate FOOBAR FOOBAR. 
  [While documenting (@TEST MGL-PAX:SECTION)]
```
Tested on current master.

**Comment by melisgl** [comment]
Thank you. Fixed in f7a6c51b.

**Comment by zellerin** [comment]
Thanks.
